src/network/network.hpp

```diff
@@ -53,6 +53,14 @@ class Network {
   bool pbft_syncing();
   uint64_t syncTimeSeconds() const;
 
+  uint64_t pbftSyncingPeriod() const;
+  void syncBlockQueuePop();
+  std::shared_ptr<std::pair<SyncBlock, dev::p2p::NodeID>> processSyncBlock();
```

Comment:
Why does it return shared_ptr ? I dont see any copies that would justify using pointer. If it is here to check if there is some return value, let's use std::optional